### ISSUE TITLE
Feature Request for Windows: Let Nextcloud Appear in Microsoft Office "Open" and "Save as" dialogues

### ISSUE THREAD

**Issue by steffen-moser** [nextcloud/desktop#6878]
Microsoft Office 2021 Professional (and other Microsoft Office versions and variants) allows the addition of cloud storage services to fast access in the "Open" and the "Save As" dialogues:
![Nextcloud MS Office Integration 01](https://github.com/nextcloud/desktop/assets/4097996/eba901c1-0d60-4730-bd9a-d93514252e91)

![Nextcloud MS Office Integration 02](https://github.com/nextcloud/desktop/assets/4097996/cd864fbd-1e33-4d98-a1e7-25f9e8aff385)
Commercial cloud storage clients (both, Microsoft services like _OneDrive_ and Non-Microsoft services like _Dropbox_) register themselves and appear in these dialog boxes. I find it useful if Nextcloud could do the same after installing Nextcloud Desktop Client on Windows.

More information about this feature can be found at Microsoft.com: https://techcommunity.microsoft.com/t5/storage-at-microsoft/adding-work-folders-to-office-locations/ba-p/425617

**Comment by radiotech655** [comment]
Has there been any movement on this issue?

**Comment by Phyroks** [comment]
This would be an Amazing Feature

**Comment by tdrieshof** [comment]
Second that. Too many clicks now to use it in an enterprise environment...

**Comment by qistoph** [comment]
I support this feature request and have made an initial list of requirements based on https://www.microsoft.com/en-us/download/details.aspx?id=35474.

- [ ] Cloud storage service must sync to a local folder (Note: The cloud service should update the LocalFolderRoot registry value if the physical location of the root folder is changed.)
- [ ] The service provider should be identified by the same GUID on all client installations.
- [ ] Display Name - Display name for the service (Suggestion: "Nextcloud").
- [ ] Description - Description string for the service provider. (Suggestion: "Store documents on your Nextcloud instance.")
- [ ] Url to 48x48 pixel thumbnail
- [ ] LearnMoreURL - URL to a web landing page where users can learn more and download the client installer.

**Comment by qistoph** [comment]
Using `regedit` and manually settings some of the options I had to try it out. I can't get the integration to show up in the 'Add a Place' dialog of my Office 365 installation though.

```
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\Common\Cloud Storage\310CA6E3-E2F9-4BB0-B927-1D2D574C59B3]
"DisplayName"="Nextcloud"
"Description"="Store documents on your Nextcloud instance."
"Url48x48"="https://nextcloud.my-instance.com/core/img/logo/logo-icon-175px.png"
"LearnMoreURL"="https://nextcloud.my-instane.com/"
"LocalFolderRoot"="C:\\Users\\Piet\\Nextcloud"
```

**Comment by realDayaa** [comment]
The article mentions Office 2013. I don't think this can be achieved in 365, unfortunately.
I've also stumbled upon this [Microsoft forum thread](https://learn.microsoft.com/en-us/answers/questions/113031/how-to-add-my-product-to-add-a-place-in-office365) where the limitations are discussed.

tl;dr Adding custom providers doesn't seem possible at the moment.

**Comment by steffen-moser** [comment]
Hm... I don't understand how, for example, Dropbox is doing it:

<img width="1188" height="1452" alt="Image" src="https://github.com/user-attachments/assets/eb259b04-45f2-492f-9c7c-46e38c59f3b9" />
